.env

```diff
@@ -37,6 +37,8 @@ HASH_REALTIME_PORT=3333
 HASH_REDIS_HOST=localhost
 HASH_REDIS_PORT=6379
 
+HASH_OTLP_ENDPOINT=http://localhost:4317
```

Comment:
Do we need to do anything to disable this for the graph? (As in does the `.env` override the docker compose stuff, so if we drive-by comment out the env-var in the docker compose is this just going to re-enable it again?)

Reply:
The environment variables passed to docker compose are all explicit, and won't be passed to the containers unless specified in the `env` key. The variable names are also different, the one for the graph is `HASH_GRAPH_OTLP_ENDPOINT` and not `HASH_OTLP_ENDPOINT`, so we should be fine in that regard :)